README.md

```diff
@@ -173,6 +173,10 @@ print(str(r))
 
 The `Dockerfile` present in this repository and its respective `twilio/twilio-python` Docker image are currently used by Twilio for testing purposes only.
 
+### Using a Custom HTTP Client
+
+To use a custom HTTP client with this helper library, please [check the Twilio documentation](https://www.twilio.com/docs/libraries/python/custom-http-clients-python).
```

Comment:
Link should be consistent with other PRs: `please see the [Twilio documentation]`